zombienet_tests/README.md

```diff
@@ -12,6 +12,35 @@ _The content of this directory is meant to be used by Parity's private CI/CD inf
 * [zombienet repo](https://github.com/paritytech/zombienet)
 * [zombienet book](https://paritytech.github.io/zombienet/)
 
+## Running tests locally
+
+To run any test locally use the native provider (`zombienet test -p native ...`) and set `ZOMBIENET_NATIVE_BINARY_PREFIX` environment variable with a full path to a directory containing polkadot/cumulus binaries. E.g. `ZOMBIENET_NATIVE_BINARY_PREFIX=/home/user/local-run/`. **The path should contain a trailing slash!**
```

Comment:
Can we not remove the trailing slash?

Reply:
(By changing the zombienet tests?)

Reply:
Actually we can do this in a much simpler way. Instead of hardcoding the path in the test file we can just modify the PATH variable for the execution. E.g.:

```
$ PATH=/home/user/local-run/:$PATH zombienet test -p native 0001-parachains-pvf.zndsl
```
If I'm not mistaken, binaries are looked up in each directory, in the order they are listed in PATH, so it will work just fine.

Reply:
I worded the last sentence in an awful way but you understand what I mean.

Reply:
Probably depends on how zombienet is implemented? But if you already have some custom env variable, why trying to build in some second way? And if we follow your thinking, we could remove the custom env variable directly and people could modify the PATH only.

Reply:
Actually this is what I'm suggesting - I removed ZOMBIENET_NATIVE_BINARY_PREFIX altogether.

> But if you already have some custom env variable, why trying to build in some second way?

I don't understand this. What do you mean with 'build in some second way'?

Reply:
Ignore what I said. 🙈 I have now seen your latest changes! Looking good. 👍